### PR TITLE
Fixing up tests 03, 06, 14, and 290 for the VCK5000

### PR DESCRIPTION
**Pull request by eddierichter-amd**

### CODE REVIEW

File: test/26_air_beefmaker_extfn/test.cpp [hunk 1]
```diff
@@ -1,203 +1,103 @@
-//===- test.cpp -------------------------------------------------*- C++ -*-===//
+//===- main.cpp -------------------------------------------------*- C++ -*-===//
```
Comment (fifield) [inline test/26_air_beefmaker_extfn/test.cpp:1]: the file is test.cpp

File: test/26_air_beefmaker_extfn/test.cpp [hunk 2]
```diff
@@ -1,203 +1,103 @@
...
+#include <stdlib.h>
+#include <sys/time.h>
+#include <vector>
+
+#define VERBOSE 1
+#define PROFILE 0
+
+/*namespace air::partitions::partition_0 {
+int32_t mlir_aie_read_buffer_beef_0_0(aie_libxaie_ctx_t *ctx, int index);
+} // namespace air::partitions::partition_0*/
```
Comment (fifield) [inline test/26_air_beefmaker_extfn/test.cpp:23]: remove this

**Review by fifield — APPROVED** [review]

File: test/26_air_beefmaker_extfn/test.cpp (hunk 2 above)
Comment (eddierichter-amd) [reply test/26_air_beefmaker_extfn/test.cpp]: Good catch. Addressed in [f67896e](https://github.com/Xilinx/mlir-air/pull/266/commits/f67896e1d6f1ff6dfccad61e562540cce8725c98).

File: test/26_air_beefmaker_extfn/test.cpp (hunk 1 above)
Comment (eddierichter-amd) [reply test/26_air_beefmaker_extfn/test.cpp]: Good catch. Addressed in [f67896e](https://github.com/Xilinx/mlir-air/pull/266/commits/f67896e1d6f1ff6dfccad61e562540cce8725c98).

**Pull request merged**